tox.ini

```diff
@@ -1,11 +1,11 @@
 [tox]
-envlist = py36, py37, {py36,py37}-no-ext, lint, check, security, docs
+envlist = py36, py37, pyNightly, {py36,py37,pyNightly}-no-ext, lint, check, security, docs
```

Comment:
Can we restrict the `nightly` build check/support only under `travis`? For `tox` it's better to have a fixed version that developers can use. Nightly might not always work and can mark tests for failure.

Reply:
Is there any reason to have any different local testing environments with `py36` vs `py37`?

Would it be better to just change this to `py3` and let travis handle installing the separate python versions to test.